CoreVulkanDisplay/src/au/gov/asd/tac/constellation/visual/vulkan/CVKVisualProcessor.java

```diff
@@ -232,7 +232,8 @@ protected void initialise() {
     @Override
     protected void cleanup() {
         // TODO_TT:
-        //canvas.removeGLEventListener(renderer);
+        cvkRenderer.Destroy();
```

Comment:
Can probably remove the comments above and below.  It would be good if you could dig into the destruction a bit.  See if you can find a way to spot memory leaks as we almost certainly have graph resources we aren't releasing.

Reply:
Removed old comments